docs/submission-getting-started.md

```diff
@@ -4,9 +4,9 @@ title: Getting Started
 ---
 
 
-After receiving your data submitter program invitation email to the ICGC ARGO Data Platform (ARGO Platform) from the Data Coordination Center (DCC), you are ready to submit clinical and molecular data! Submission of data can start as soon as you join your program.
+After receiving your data submitter program invitation email to the ARGO Data Platform from the Data Coordination Center (DCC), you are ready to submit clinical and molecular data! Submission of data can start as soon as you join your program.
 
-## Submitting Data to the ARGO Platform
+## Submitting Data to the ARGO Data Platform
```

Comment:
this is pretty ugly  with the double Data stated.  should we say `Submitting to the ARGO Data Platform`  instead ?

Reply:
yes!